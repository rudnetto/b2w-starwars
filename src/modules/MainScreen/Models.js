// @flow
type Planet = {
  name: string;
  population: number;
  climate: string;
  terrain: string;
  filmsFeatured: number;
};

export default {
  Planet,
};
