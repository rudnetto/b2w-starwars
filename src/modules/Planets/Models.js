// @flow
export type Planet = {
  name: string;
  population: number;
  climate: string;
  terrain: string;
  filmsFeatured: number;
};
