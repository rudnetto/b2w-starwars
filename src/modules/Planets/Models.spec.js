import Models from './Models';

describe('Models test suite', () => {
  it('should preserve structure', () => {
    expect(Models).toMatchSnapshot();
  });
});
