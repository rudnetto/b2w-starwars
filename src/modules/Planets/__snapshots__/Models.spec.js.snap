// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Models test suite should preserve structure 1`] = `Object {}`;
